libs/diag/diagnostic.go

```diff
@@ -96,7 +99,14 @@ func (ds Diagnostics) HasError() bool {
 func (ds Diagnostics) Error() error {
 	for _, d := range ds {
 		if d.Severity == Error {
-			return fmt.Errorf(d.Summary)
+			message := d.Detail
```

Comment:
Are there cases where the summary isn't set?

I'd like to have detail be optional and summary required.

Reply:
I'm pretty sure `Summary` is always set. This code is just testing if `Detail` is set.